### DISCUSSION TITLE
Please document HasAttributes->getDirty() on Eloquent Models in the Laravel docs

### DISCUSSION THREAD

**Discussion by HenkPoley** [laravel/framework#53617] [General] [answered]
Seems to be pretty old functionality; I find it locally in a stock Laravel 5.4, maybe older.

It is not in the main docs.

**Comment by henzeb** [comment, marked as answer]
You can do it your self. The doc is maintained by the community

**Reply by shaedrich** [reply]
@HenkPoley Feel free to submit a PR at [`laravel/docs`](https://github.com/laravel/docs), although chances for it being rejecting are generally high

**Reply by HenkPoley** [reply]
@shaedrich I've tried here: https://github.com/laravel/docs/pull/10044

**Reply by ghost** [reply]
@shaedrich is this a habit to reject positive gestures or why?

**Reply by shaedrich** [reply]
@HenkPoley Fingers crossed! 🤞🏻 
@marius-mcp You should ask Taylor about his habits not me 🤷🏻‍♂️

**Reply by ghost** [reply]
@shaedrich I don't need to ask him, if you know what I mean...

**Reply by shaedrich** [reply]
Touché 😂

**Reply by HenkPoley** [reply]
I think my current attempt is a bit verbose. So I guess they'll reject it. I think it would be best to weave the whole isDirty()/getDirty()/etc. together in one short text.

I've previously been rejected because it didn't go to the oldest supported branch (then I forgot/couldn't be bothered to re-submit).

Also I had attempted to fix the link at the top of older documentation that always goes to the _main page_ of Laravel latest (e.g. 11.x now) instead of the same documentation page of 11.x. Closed as "I'll think about it". To be fair my attempt was broken because I didn't understand the documentation parser, but still not useful.

**Reply by shaedrich** [reply]
"I'll think about it" is more than I get most of the time, though there have been rare exceptions.
